netty/shaded/build.gradle

```diff
@@ -46,14 +46,6 @@ shadowJar {
     mergeServiceFiles()
 }
 
-// This is a hack to have shadow plugin modify the uploadArchives POM's
```

Comment:
grpc-netty-shaded's pom doesn't look right. It shouldn't have dependencies on netty and tcnative.

Reply:
discussed offline. NOTE for others: because shadow plugin is filtering project deps, we had to manually generate dependencies.